Cargo.toml

```diff
@@ -2,6 +2,8 @@
 members = [
     "src/backend",
     "src/cycles_ledger/client",
+    "src/cycles_ledger/pic",
+    "src/cycles_ledger/types",
     "src/shared"
```

Comment:
Note:
- `backend` and `shared` are manually written crates.
- `client` is generated by `didc` and then manually modified; that manual modification will be automated in a future PR.
- `pic` and `types` are the new crates with type definitions and pocket-ic bindings.

All the code below is the boilerplate code needed to add these new crates.